src/common/exif.cc

```diff
@@ -1378,6 +1378,7 @@ int dt_exif_read_blob(uint8_t *buf, const char *path, const int imgid, const int
       memcpy(buf + 6, &(blob[0]), length);
       return length + 6;
     }
+    fprintf(stderr, "[exif] Warning: too much data to store (%d bytes), Exif data will be discarded\n", length);
```

Comment:
Actually, it's a static array. Maybe this could be allocated with more space, as darktable will not inflate the array by too much, or switch to malloc'ed arrays...